.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+<!-- This comments are hidden when you submit the issue, so you do not need to remove them! -->
```

Comment:
You need to add the following metadata to the top of the file.

```
---
name: Bug report
about: Create a report to help us improve

---
```

Same with feature request.

Reply:
Done!

Reply:
"Create a report describing a bug in SunPy"